data/channels.csv

```diff
@@ -7826,7 +7826,7 @@ HistoryHDCanadaEast.ca,History HD Canada East,,,CA,,,c/CA,eng,documentary,FALSE,
 HistoryHDCanadaWest.ca,History HD Canada West,,,CA,,,c/CA,eng,documentary,FALSE,2019-09-18,,,https://www.history.ca/,https://upload.wikimedia.org/wikipedia/commons/thumb/0/01/History_%282021%29.svg/512px-History_%282021%29.svg.png
 HistoryHDChile.cl,History HD Chile,,,CL,,,c/CL,spa,documentary,FALSE,,,,https://www.historyplay.tv/,https://upload.wikimedia.org/wikipedia/commons/thumb/0/01/History_%282021%29.svg/512px-History_%282021%29.svg.png
 HistoryHDGermany.de,History HD Germany,,,DE,,,c/DE,deu,documentary,FALSE,,,,http://www.history.de/,https://upload.wikimedia.org/wikipedia/commons/thumb/0/01/History_%282021%29.svg/512px-History_%282021%29.svg.png
-HistoryHDHungary.hu,History HD Hungary,,,HU,,,c/HU,hun,documentary,FALSE,,,,http://www.history.com/,https://upload.wikimedia.org/wikipedia/commons/thumb/0/01/History_%282021%29.svg/512px-History_%282021%29.svg.png

```

Comment:
Why remove this channel?

Reply:
It seems like it was made as a modification, not even as an addition.

Reply:
> Why remove this channel?

Should be fixed now.